lib/community/widgets/community_sidebar.dart

```diff
@@ -342,10 +342,7 @@ class BlockCommunityButton extends StatelessWidget {
             child: Row(
               mainAxisAlignment: MainAxisAlignment.center,
               children: [
-                Icon(
-                  blocked ? Icons.undo_rounded : Icons.block_rounded,
-                  semanticLabel: (state.communityView?.blocked ?? communityView.blocked) ? 'Unblock Community' : 'Block Community',
```

Comment:
I removed a bunch of semantic labels that are redundant since the text already gets read from the `Text` widget.